### PR TITLE
Resultserver rotation test timing

### PR DESCRIPTION
**Pull request by mrogers950**
The last resultserver unit test (rotation of 3 directories with policy=2) often failed on ppc64le systems. It was observed that test directories created only within 5 milliseconds of each other could result in an identical creation time, and affect the expected results.

This increases the delay between test directories to 100ms to better ensure a different creation time on ppc64le.

### CODE REVIEW

File: cmd/manager/resultserver_test.go [hunk 1]
```diff
@@ -154,7 +154,6 @@ var _ = Describe("Resultserver testing", func() {
 			Expect(err).To(BeNil())
 
 			files := _readDirNames(rootDir)
-
```
Comment (rhmdnd) [inline cmd/manager/resultserver_test.go:157]: Unrelated whitespace change?

**Review by rhmdnd — COMMENTED** [review]
Looks good to me. Waiting on a clean CI run.

File: cmd/manager/resultserver_test.go (hunk 1 above)
Comment (mrogers950) [reply cmd/manager/resultserver_test.go]: woops, fixed

**Review by JAORMX — APPROVED** [review]
/lgtm
